plugins/GSdx/Renderers/Common/GSOsdManager.cpp

```diff
@@ -357,7 +358,17 @@ size_t GSOsdManager::GeneratePrimitives(GSVertexPT1* dst, size_t count) {
 		float offset = 0;
 
 		for(auto it = m_log.begin(); it != m_log.end();) {
-			float x = -1 + 8 * (2.0f/m_real_size.x);
+			// This is for the osd monitor location pr;
+			// Whenever the monitor osd is set to top left, move the logs to the top right
+			float x;
```

Comment:
This too.

Reply:
It doesn't matter. There are only two possible cases and it get initialized in both. You could initialize it to 0 or something, but functionally it does nothing.

another option that might be cleaner:
```cpp
float x = -1.0f + 8 * (2.0f / m_real_size.x);

if(!(m_monitor_enabled && monitor_pos != GSOSD_POS::TOPLEFT))
    x -= StringSize(it->msg);
```

or however you want to invert that if

Reply:
I suggested it so VS can stop complaining.

Reply:
The suggestion above has been pushed.